pilot/pkg/xds/gen.go

```diff
@@ -118,6 +118,12 @@ func (s *DiscoveryServer) pushXds(con *Connection, push *model.PushContext,
 		Resources:    res,
 	}
 
+	configSize := ResourceSize(res)
+	configSizeBytes.With(
+		typeTag.Value(w.TypeUrl),
+		namespaceTag.Value(con.proxy.ConfigNamespace),
```

Comment:
is namespace too high cardinality? some folks even have transient namespaces coming up and down. We don't have this as a tag for other metrics

Reply:
Potentially, I don't generally think of namespace as being _that_ dynamic, but if that is cycling quickly it could be an issue.

Is there still a desire to know per-namespace config limits?

Reply:
I don't personally know of any? I think if you need that granularity you can check out the pods and see exactly which pod is getting big responses?

The metric would allow you to alert on config sizes being large, but can't give fine grained details

Reply:
Removed.